test/end-to-end/signup.test.js

```diff
@@ -414,3 +415,134 @@ describe("SuperTokens SignUp => Server Error", function () {
         await waitForSTElement(page, "[data-supertokens~=generalError]", true);
     });
 });
+
+// CUSTOM FIELDS TEST
+
+describe("Signup custom fields test", function () {
+    let browser;
+    let page;
+    let consoleLogs;
+
+    before(async function () {
+        await backendBeforeEach();
+
+        await fetch(`${TEST_SERVER_BASE_URL}/startst`, {
+            method: "POST",
+        }).catch(console.error);
+
+        browser = await puppeteer.launch({
+            args: ["--no-sandbox", "--disable-setuid-sandbox"],
+            headless: false,
+        });
+        page = await browser.newPage();
+    });
+
+    // after(async function () {
+    //     await browser.close();
+    // });
+
+    afterEach(function () {
+        return screenshotOnFailure(this, browser);
+    });
+
+    beforeEach(async function () {
+        consoleLogs = [];
+        consoleLogs = await clearBrowserCookiesWithoutAffectingConsole(page, consoleLogs);
```

Comment:
given that we are adding this in the top level describe, which already has this, do we also need to do this here again?

Reply:
now, using the top-level hooks